code/modules/events/wallrot.dm

```diff
@@ -1,9 +1,5 @@
 /datum/event/wallrot/setup()
```

Comment:
Can be removed.
The event only has to start. Then it can be killed again. It would be a waste to have it tick while not doing anything.
If both `startWhen` and `endWhen` are 0, which is the default, then it'll only call `start` and then afterwards instantly call `end`

Reply:
Mm, I wasn't familiar enough with event code to know if it did things constantly through the lifetime of the event, or if it just did it at the start.